libs/design-system-components/src/components/ExtensionImageGallery/index.tsx

```diff
@@ -34,21 +33,27 @@ const ExtensionImageGallery: React.FC<ExtensionImageGalleryProps> = props => {
 
   return (
     <>
-      <div className={images.length === 1 ? tw('flex') : tw(`${gridStyle}`)}>
-        {images.map((image, index) => (
+      <div className={images?.length === 1 ? tw('flex') : tw(`${gridStyle}`)}>
+        {images?.map((image, index) => (
           <ImageBlockGridItem
-            image={image}
+            image={{ src: image.src, size: { width: image.width, height: image.height } }}
             key={index}
             handleClickImage={handleClickImage}
-            images={images}
+            images={images.map(img => ({
```

Comment:
shouldn't here be also optional chaining for `?.map`?

Reply:
I didn't see the shadowing, so every ImageBlockGridItems receives the entire images list?